cms/signals/page.py

```diff
@@ -3,10 +3,11 @@
 from cms.signals.apphook import set_restart_trigger
 
 
-def pre_save_page(instance, **kwargs):
+def pre_save_page(instance, raw=False, **kwargs):
```

Comment:
raw is passed as kwargs so we can leave it out

Reply:
fixed